docs/src/tutorials/bseries_basics.md

```diff
@@ -0,0 +1,163 @@
+# [B-Series basics](@id tutorial-bseries-basics)
+In this tutorial we use `bseries.jl` to investigate error expansions for RK
+methods, generically or when applied to a specific ODE.
+
+
+```@example bseries-basics
+# Load the packages we will use.  These must first be installed using: import Pkg; Pkg.add("package_name")
+using BSeries
+using Latexify  # Only needed for some pretty-printing cells below
+import SymPy; sp=SymPy;
+```
+
+# B-Series for a generic ODE
+
+First we specify the Butcher coefficients of the RK method.
+This can include symbolic expressions and parameterized families of methods.
+Here is a generic 2-stage, 2nd-order method:
+
+
+```@example bseries-basics
+α = sp.symbols("α", real=true)
+A = [0 0; 1/(2*α) 0]; b = [1-α, α]; c = [0, 1/(2*α)]
+coeffs2 = bseries(A,b,c,3)
+latexify(coeffs2, cdot=false)
+```
+
+We have generated the B-series up to terms of order $h^3$.  The terms $F_f()$
+represent elementary differentials, which are products of derivatives of the
+ODE right-hand side.  Since we haven't specified an ODE, these are indicated
+simply by the associated rooted tree.  The rooted trees are printed as nested
+lists, essentially in the form used in Butcher's book.  The rooted trees written
+in this way can be rendered in LaTex using the package `forest`; unfortunately,
+there is no easy way to render them in the browser.
+
+Here is a B-series for a 4th-order method, expanded up to 5th-order terms:
+
+
+```@example bseries-basics
+A = Rational{Int128}[0 0 0 0 0 0 0 0;(-1//6) (1//2) 0 0 0 0 0 0;(-1//10) (1//10) (1//2) 0 0 0 0 0;(-21463//39375) (21017//26250) (-5//9) (1//2) 0 0 0 0;(-59588//54675) (118717//36450) (-4375//2187) 0 (1//2) 0 0 0;(-19993033//9443328) (28508695//3147776) (-13577105//2360832) (-4090625//3147776) (1136025//3147776) (1//2) 0 0;(367020141781//199294617600) (814214904871//22143846400) (-29834937659//1992946176) (-1983358776875//87689631744) (-6702625935//885753856) (688576//109395) (1//2) 0;(1081252805//134140608) (2639189439//74522560) (33646441//4191894) (-7873511875//210792384) (-504040617//14904512) (2110843561//115277085) (13//7) (1//2)];
+b = Rational{Int128}[(1081252805//134140608),(2639189439//74522560),(33646441//4191894),(-7873511875//210792384),(-504040617//14904512),(2110843561//115277085),(13//7),(1//2)];
+c = Rational{Int128}[0,(1//3),(1//2),(1//5),(2//3),(3//4),(1//4),1];
+
+coeffs4 = bseries(A,b,c,5)
+latexify(coeffs4, cdot=false)
```

Comment:
Maybe write the rows of `A` in different rows, not a single line? And maybe add a reference for the coefficients?

Reply:
Ah, okay. For me, both options are fine. Maybe it would be nice for a basic tutorial to use a method most people already know?

Reply:
Okay, I replaced it with the standard explicit 4th-order RK method.